client/components/google-search/google-search.js

```diff
@@ -13,13 +13,15 @@ export default class GoogleSearch extends Component {
     constructor(props) {
         super(props);
         this.state = {
-            short_name: '',
-            placeId: '',
+            streetName: '',
+            place_id: '',
             location: {
                 lat: null,
                 lng: null,
             },
         };
+
+        this.handleSearchClicked = this.handleSearchClicked.bind(this);
```

Comment:
This shouldn't be necessary as `handleSearchClicked` is implemented using an arrow function. Arrow functions pass `this` as context by default.